linkerd/proxy/http/src/version.rs

```diff
@@ -34,11 +36,15 @@ impl Version {
             // We didn't want to keep parsing headers, just validate that
             // the first line is HTTP1.
             Ok(_) | Err(httparse::Error::TooManyHeaders) => {
+                trace!("Detected H1");
                 return Some(Version::Http1);
             }
             _ => {}
         }
 
+        debug!("Not HTTP");
+        trace!(?bytes);
+        trace!(h2.preface = ?Self::H2_PREFACE);
```

Comment:
nit, take it or leave it: any reason this isn't
```suggestion
        trace!(?bytes, h2.preface = ?Self::H2_PREFACE);
```

Reply:
it's easier to compare then when they're not on one long line.

Reply:
ah, fair enough.